### DISCUSSION TITLE
Meeting Tomorrow

### DISCUSSION THREAD

**Discussion by ExtremeMachine12** [tanmaypardeshi/CSC-510-Project3-PopcornPicks#5] [General]
Hey, y'all let's meet up tomorrow to work on the project in person.

**Comment by tanmaypardeshi** [comment]
Sure. The time decided for the meeting is 4:30pm right?

**Comment by tanmaypardeshi** [comment]
Good meeting with everyone! Let's get the project done before the deadline.
